math_series/series.py

```diff
@@ -0,0 +1,39 @@
+def fibonacci(n):   
+    if n == 0: 
+        return 0
+    elif n == 1 or n == 2: 
+        return 1
+    else:
+        return n * fibonacci((n - 1 ) + (n - 2))
```

Comment:
Remember that for these series functions, we are adding the previous results to get the current result. Instead of the implementation you have on lines 7 and 15, consider something closer to:

```
return (previous result) + (previous previous result)
```